AzureCalling/Controllers/LobbyViewController.swift

```diff
@@ -14,8 +14,7 @@ class LobbyViewController: UIViewController, UITextFieldDelegate {
     var callingContext: CallingContext!
     var previewRenderer: VideoStreamRenderer?
     var rendererView: RendererView?
-    var joinInput: String?
-    var joinCallType: JoinCallType = .groupCall
```

Comment:
We can keep the `joinInput` and `joinCallType`, but only have one type for now. There will be less conflicts when we merge later on.